lib/Dialect/TTNN/Transforms/TTNNToSerializedBinary.cpp

```diff
@@ -151,6 +151,18 @@ createReductionOp(FlatbufferObjectCache &cache, ReductionOp op) {
                                                dim_arg, op.getKeepDim());
 }
 
+template <typename SoftmaxOp>
+::flatbuffers::Offset<::tt::target::ttnn::SoftmaxOp>
+createSoftmaxOp(FlatbufferObjectCache &cache, SoftmaxOp op) {
+  auto in =
```

Comment:
I prefer correctly naming types instead of auto unless it's some kind of iterator or some similar kind of untrivial type. Makes code more readable especially during learning phase of new project/framework.

Reply:
After changing from auto to concrete type, it seems a bit unreadable, hence I will revert back to auto:
![image](https://github.com/user-attachments/assets/4e611eae-3677-4685-8d1f-721cfd10fa98)

Reply:
Changed dimension return type from auto to int32_t